exercises/proverb/.meta/gen.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"log"
+	"text/template"
+
+	"../../../gen"
+)
+
+func main() {
+	t, err := template.New("").Parse(tmpl)
+	if err != nil {
+		log.Fatal(err)
+	}
+	var j js
+	if err := gen.Gen("proverb", &j, t); err != nil {
+		log.Fatal(err)
+	}
+}
+
+// The JSON structure we expect to be able to unmarshal into
+type js struct {
+	Cases []struct {
+		Description string
+		Cases       []struct {
+			Description string
+			Property    string
+			Input       struct {
+				Phrase string
```

Comment:
From looking at the problem-specifications/exercises/proverb/canonical-data.json, the
`       Phrase string`
should instead be
`       Strings []string`
to properly parse.